### PR TITLE
add additional description 

### PR DESCRIPTION
**Pull request by zohra-anwar**
add description for tag, name, body,  prerelease, overwrite, artifacts,

### CODE REVIEW

**Comment by majkrzak** [comment]
It took me a while to figure out what the hell 😂   @andrew-kulpa can you explain yourself?

**Review by majkrzak — CHANGES_REQUESTED** [review]
Thank you for the contribution, but before I will merge your changes you have to rebase your branch to a84d1391af8b298fc1327007fb94ed1211362013 and also describe parameters with `description` key instead of comments.

**Review by aapokiiso — APPROVED** [review]

**Comment by NikoGrano** [comment]
There is some changes I would like comment t about. Review. Coming in a few hours.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules*
```
Comment (NikoGrano) [inline .gitignore:1]: => /node_modules, not `node_modules/*`

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2019 Andrew Kulpa
```
Comment (NikoGrano) [inline LICENSE:1]: This name should be changed.

File: README.md [hunk 3]
```diff
@@ -1,20 +1,41 @@
-majkrzak/create-release
+Andrew-Kulpa/create-release
```
Comment (NikoGrano) [inline README.md:1]: Name here should be changed

File: README.md [hunk 4]
```diff
@@ -1,20 +1,41 @@
...
+```
+
+
 Example
 -------
 
 ```yaml
-    - uses: majkrzak/create-release@master
+    - name: Release Executables
+      uses: andrew-kulpa/create-release@v1
```
Comment (NikoGrano) [inline README.md:28]: Name here.

File: README.md [hunk 5]
```diff
@@ -1,20 +1,41 @@
...
       with:
-        token: ${{github.token}}
-        name: My lovely release
-        code: latest
-        prerelease: true
-        assets: >
-          source.txt:target.txt:text/plain
-          another:one:application/json
+        name: Your release name
```
Comment (NikoGrano) [inline README.md:30]: Might be better to provide linux and windows both as example. Mostly stuff are begin built on linux.

File: action.yml [hunk 6]
```diff
@@ -1,25 +1,52 @@
-name: majkrzak/create-release
+name: Andrew-Kulpa/create-release
```
Comment (NikoGrano) [inline action.yml:1]: Name here.

File: action.yml [hunk 7]
```diff
@@ -1,25 +1,52 @@
...
 description: |
-  Github Action for handling release creation
-author: Piotr Majkrzak
+  Github Action for Uploading Release Artifacts
```
Comment (NikoGrano) [inline action.yml:3]: And here.

File: package.json [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+    "main": "lib/main.js",
+    "scripts": {
+        "build": "tsc"
+    },
+    "keywords": [
+        "actions",
+        "github",
+        "release"
+    ],
+    "author": "Andrew-Kulpa",
```
Comment (NikoGrano) [inline package.json:14]: Change

**Review by NikoGrano — CHANGES_REQUESTED** [review]
Would recommend to change following.

@majkrzak I would highly recommend to make eslint config which could be applied here. There is some code style related stuff, which would just more simply get fixed by applying eslint.

@zohra009 Thank you for you contribution!

File: lib/main/main.js [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+            }
+        }
+    });
+}
+function createRelease(body, tag, prerelease, name) {
+    return __awaiter(this, void 0, void 0, function* () {
+        const release = yield api.repos.createRelease(Object.assign(Object.assign({}, github.context.repo), { tag_name: tag || 'DEFAULT_RELEASE', target_commitish: github.context.sha, name: name || tag || 'DEFAULT_RELEASE', body: body, prerelease: prerelease }));
+        return release.data; // {upload_url, html_url, ...}
+    });
+}
```
Comment (Arhimenrius) [inline lib/main/main.js]: Also, transpiled files shouldn't be placed in the repository.

**Review by Arhimenrius — COMMENTED** [review]
Please never put transpiled files to repository.

File: lib/main/main.js [hunk 10]
```diff
@@ -0,0 +1,109 @@
+"use strict";
```
Comment (Arhimenrius) [inline lib/main/main.js:1]: Transpiled code should never appear in repository.

**Review by Arhimenrius — CHANGES_REQUESTED** [review]
It's about previous comment. Basically entire lib/main/main.js should be taken away.

**Comment by majkrzak** [comment]
After detailed review I see that this PR qualifies to spam.

**Pull request closed without merging**